### ISSUE TITLE
No 'Action' options showing in Automation

### ISSUE THREAD

**Issue by PeterChia604** [qqaatw/JciHitachiHA#11]
On fresh install, unable to select 'Actions' for automation routines.
The pull down tab is greyed out and has no option to select.
Running Rspberry OS64bit, running Home Assistant 2022.2.9

**Comment by qqaatw** [comment]
Would be helpful if you provide the model of your device and screenshots that show the greyed out pull down menu.

**Comment by qqaatw** [comment]
FYI, mine has options in the action pull down menu when the selected action type is `Device`

**Comment by PeterChia604** [comment]
My bad.  Sorry. 
I had Emporia Vue (for Power measurement) and the sensor had the same name.  It is a sensor, so of course, no 'Actions' can be performed.    My mistake.  I didn't even notice until I scrolled down further I had two 'devices' with the same name.
Thanks for the feedback.  :-)
btw, how do I set the temperature, swing mode, fan speed, etc? 

![Actions 1](https://user-images.githubusercontent.com/95140180/155751598-dbc8b253-1b49-4423-9d9f-7c4731e27250.png)
![Actions 2](https://user-images.githubusercontent.com/95140180/155751743-2ba59b95-4145-4256-acd3-729def15d6db.png)
